controllers/configurationpolicy_controller.go

```diff
@@ -1344,7 +1364,8 @@ type templateIdentifier struct {
 
 // createInformStatus updates the status field for a configurationpolicy with remediationAction=inform
 // based on how many compliant/noncompliant objects are found when processing the templates in the configurationpolicy
-func createInformStatus(
+// Or multiple namespaces and enforce use this
+func createMergedStatus(
```

Comment:
The function name in the description should also match the new function name.

Reply:
good good good Thanks